sdk/cosmos/azure-cosmos-encryption/src/samples/java/com/azure/cosmos/AApProgram.java

```diff
@@ -0,0 +1,345 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.cosmos.encryption.AAPContainer;
+import com.azure.cosmos.encryption.AlwaysEncrypted;
+import com.azure.cosmos.encryption.AzureKeyVaultKeyStoreProvider;
+import com.azure.cosmos.encryption.AzureKeyVaultKeyWrapMetadata;
+import com.azure.cosmos.encryption.ClientEncryptionIncludedPath;
+import com.azure.cosmos.encryption.ClientEncryptionPolicy;
+import com.azure.cosmos.encryption.CosmosDataEncryptionKeyProviderMetadata;
+import com.azure.cosmos.encryption.CosmosEncryptionAlgorithm;
+import com.azure.cosmos.encryption.EncryptionItemRequestOptions;
+import com.azure.cosmos.encryption.EncryptionKeyWrapResult;
+import com.azure.cosmos.encryption.EncryptionType;
+import com.azure.cosmos.implementation.apachecommons.lang.StringUtils;
+import com.azure.cosmos.implementation.encryption.AzureKeyVaultKeyWrapProvider;
+import com.azure.cosmos.implementation.encryption.DataEncryptionKeyProperties;
+import com.azure.cosmos.implementation.encryption.UserProvidedTokenCredentialFactory;
+import com.azure.cosmos.implementation.guava25.base.Preconditions;
+import com.azure.cosmos.models.CosmosItemResponse;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.cosmos.models.ThroughputProperties;
+import com.azure.identity.ClientSecretCredential;
+import com.azure.identity.ClientSecretCredentialBuilder;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.bouncycastle.util.encoders.Hex;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.security.Security;
+import java.time.Instant;
+import java.util.Properties;
+import java.util.UUID;
+
+// ----------------------------------------------------------------------------------------------------------
+// Prerequisites -
+//
+// 1. An Azure Cosmos account -
+//    https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal
+//
+// 2. Azure Key Vault setup.
+// ----------------------------------------------------------------------------------------------------------
+// Sample - demonstrates the basic usage of client-side encryption support in the Cosmos DB SDK.
+// ----------------------------------------------------------------------------------------------------------
+public class AApProgram {
+    static {
+        Security.addProvider(new BouncyCastleProvider());
+    }
+
+    public static final String databaseId = "testdb";
+    private static final String containerId = "encryptedData";
+    private static final String keyContainerId = AlwaysEncrypted.keyContainerName;
+    private static final String dataEncryptionKeyId = "theDataEncryptionKey";
```

Comment:
Since this is a public facing sample, we should follow java good practices for constant names - upper case names with _